elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -77,8 +77,8 @@ class PfeContentSet extends PFElement {
     fragment.appendChild(accordion);
 
     // Pass the theme property down to the accordion component
-    if (this.on) {
-      accordion.setAttribute("on", this.on);
+    if (this.on.value) {
```

Comment:
This was setting an object and was failing when it tried to pass that down to the accordion